spec/02-integration/05-proxy/05-dns_spec.lua

```diff
@@ -145,5 +145,41 @@ for _, strategy in helpers.each_strategy() do
         assert.response(r).has.status(503)
       end)
     end)
+    describe("udp sockets", function()
+      local domain_name = "www.example.com"
+      local address = "127.0.0.10"
+
+      lazy_setup(function()
+        local bp = helpers.get_db_utils(strategy, {
+          "services",
+        })
+
+        local fixtures = {
+          dns_mock = helpers.dns_mock.new()
+        }
+        fixtures.dns_mock:A({
+          name = domain_name,
+          address = address,
+        })
+
+        bp.services:insert {
+          name     = "foo",
+          host     = domain_name,
+        }
+
+        assert(helpers.start_kong({ database = strategy }, nil, nil, fixtures))
+      end)
+
+      lazy_teardown(function()
+        helpers.stop_kong()
```

Comment:
```suggestion
        assert(helpers.stop_kong())
```

Reply:
In my feel it's more of a programming style instead of a critical issue. Necessity for assertion against `stop_kong` means that by any chances stop_kong can't work as expected, however this's not what we expected itself.